packages/nextcloud/test/webdav_test.dart

```diff
@@ -305,7 +305,7 @@ void main() {
 
       test('Filter files', () async {
         final response = await client.webdav.put(utf8.encode('test'), PathUri.parse('filter.txt'));
-        final id = response.headers['oc-fileid']!.first;
+        final id = response.headers['oc-fileid']!.split(RegExp(r'\s*,\s*')).first;
```

Comment:
What's the purpose of this? I think you got confused by the multiple header thing. There will only ever one fileid that is returned, so you can directly take that.

Reply:
Yes I got confused. For these cases we should have used `single` over `first` to make that clear.

Reply:
True, but you need neither of those now as you can just take the complete header value.